recipes/arraylake-client/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "arraylake-client" %}
+{% set version = "0.4.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/arraylake_client-{{ version }}.tar.gz
+  sha256: e01b97f779089fde2ec3aae58e32034be8c81a93caec5e4c62a5ac0b778a68fa
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

This should fix the Windows build

```suggestion
  number: 0
  entry_points:
    - arraylake = arraylake_client.cli.main:app
```